src/rabbitmanager.js

```diff
@@ -113,6 +113,28 @@ class RabbitManager {
     vhost = encodeURIComponent(vhost);
     await this.request(`permissions/${vhost}/${user}`, {method: 'DELETE'});
   }
+
+  async queues() {
+    return await this.request('queues');
+  }
+
+  async queue(name, vhost='/') {
+    vhost = encodeURIComponent(vhost);
+    return await this.request(`queues/${vhost}/${name}`);
```

Comment:
I'll still make `name` URI encoded. It would be weird if someone could just enter an empty name or a name with non-URI compatible characters :)